presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientModule.java

```diff
@@ -97,8 +96,7 @@ public static CassandraSession createCassandraSession(
         requireNonNull(config, "config is null");
         requireNonNull(extraColumnMetadataCodec, "extraColumnMetadataCodec is null");
 
-        Cluster.Builder clusterBuilder = Cluster.builder()
-                .withProtocolVersion(ProtocolVersion.V3);
+        Cluster.Builder clusterBuilder = Cluster.builder();
```

Comment:
` select release_version from system.local where key = 'local';` sounds fine, as long as it's compatible on all of the versions we support (2.0ish through most recent)